doc/nrf/ug_contribution.rst

```diff
@@ -0,0 +1,141 @@
+.. _ug_contribution:
+
+Contributing to the |NCS|
+*************************
+
+The |NCS| is maintained by Nordic Semiconductor, but it is open to community contributions.
+
+As a general practice, we encourage to make small, controlled changes.
+This practice simplifies reviewing, merging, and rebasing.
+In addition, it keeps the change history clear and clean.
+
+When contributing to the |NCS|, you should provide as much information as you can about your change.
+Before submitting, you must also update the appropriate documentation and test your changes thoroughly.
+
+The general GitHub workflow that is used by |NCS| developers is based on a combination of Git commands on the command line and browser interaction with GitHub.
+As usual with Git, there are multiple ways of getting a task done.
+The following steps show a typical workflow:
+
+1. :ref:`dm-wf-fork`.
+   The following substeps summarize the most important commands to start working with a fork of the repository:
+
+   a. Fork the sdk-nrf repository to your personal account on GitHub.
+      To do so, click the `fork button on the sdk-nrf repository page`_ (upper right corner).
+
+   #. On your computer, change into the :file:`nrf` folder that was created when you :ref:`obtained the code <cloning_the_repositories_win>`::
```

Comment:
You are duplicating a lot of the contents you are linking to, aren't you?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_dev_model.html#fork-a-repository-of-the-ncs

Reply:
Correct.
We could remove what is repeated so users can follow referenced links instead, but personally I prefer to have the relevant information gathered on one page to avoid going back and forth between different pages.